helm/hpcc/values.schema.json

```diff
@@ -261,6 +261,24 @@
         "defaultMirrorPath": {
           "type": "string"
         },
+        "locale" : {
+          "decription": "the locale used for internationalization",
+          "type": "object",
+          "properties": {
+            "languageCode": {
```

Comment:
what are these going to be used for?
The costCurrencyLocale makes sense, but I would expect everything else to be determined by the user.

Reply:
The JavaScript functions that I looked at that handle currency formatting  requires these three components: country, language and currency (see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat.  We could of course provide the required information as 2 fields (language locale+ currency).  However, by providing the 3 required items separately, multi-language locale support is possible. i.e. JavaScript needs languageCode+countryCode+currencyCode.  C++ standard library functions require languageCode+countryCode (currency is inferred).

I moved them to the global section because I was thinking as the fields are separate, the language and country attribute could be used for other purposes.  Would we want to have a costCurrencyLocale with these 3 items under the field and quite possibly have a requirement to have another locale attribute for other purposes?

Reply:
The locale formatting may require 3 options - but I think 2 of those options are actually supplied by the client.  For instance. 
```
console.log(new Intl.NumberFormat('de-DE', { style: 'currency', currency: 'EUR' }).format(123456.789));
> "123.456,79 €"
```
which is what I would expect if I was german.
but 
```
console.log(new Intl.NumberFormat('en-UK', { style: 'currency', currency: 'EUR' }).format(123456.789));
> "€123,456.79"
```
is what I would expect as an english person viewing the same information.

i.e. the currency needs to come from the platform, but the natural way for it to be displayed really comes from the locale being used by the user.  @GordonSmith what are your thoughts?

Reply:
I agree - I plan on formatting the number per the local locale and using the short code to resolve the currency symbol and failing that use the "currency" value supplied?